mssqlcli/mssql_cli.py

```diff
@@ -150,8 +150,12 @@ def __init__(self, options):
         self.cli_style = c['colors']
         self.output_style = style_factory_output(self.syntax_style, self.cli_style)
         self.wider_completion_menu = c['main'].as_bool('wider_completion_menu')
+
+        # override interactive mode to false if query is set
+        self.interactive_mode = options.interactive_mode and not options.query
```

Comment:
I don't think we should have the `and not options.query ` as part of the condition - we should just accept what's passed.